Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/ActionsLibraryNavPage.xaml.cs

```diff
@@ -170,6 +197,9 @@ private ObservableList<Act> GetPlatformsActions(bool ShowAll = false)
                 if (a.IsSelectableAction == false)
                     continue;
 
```

Comment:
does GetPlatformsActions() is duplication to what we have in AddActionPage?

Reply:
Yes Meni, it's the copy.